pyproject.toml

```diff
@@ -0,0 +1,15 @@
+[tool.black]
+extend-exclude = '''
+/(
+  | migrations
+)/
+'''
+
+[tool.isort]
+skip = [
+  "migrations"
+]
```

Comment:
Took the liberty to exclude migrations from linting, because I don't think it makes sense to do that on auto-generated files. WDYT?

Reply:
The migrations files are not auto generated

Reply:
Alright, that's good to know!